scripts/single-node.sh

```diff
@@ -20,11 +20,9 @@ celestia-appd collect-gentxs
 # Set proper defaults and change ports
 # If you encounter: `sed: -I or -i may not be used with stdin` on MacOS you can mitigate by installing gnu-sed
 # https://gist.github.com/andre3k1/e3a1a7133fded5de5a9ee99c87c6fa0d?permalink_comment_id=3082272#gistcomment-3082272
-sed -i'.bak' 's#"tcp://127.0.0.1:26657"#"tcp://0.0.0.0:26657"#g' ~/.celestia-app/config/config.toml
```

Comment:
[not for reviewers]
this was moved to the `client.toml` file

Reply:
Apparently sometime between `v0.12.0-rc7`  and latest master. I checked the history and it seems that only this https://github.com/celestiaorg/celestia-app/commit/990c7201c395551e56652b086056e05fe4150763 touches on those. Want me to investigate?

Reply:
I compiled main and reintiated, and for me I'm still seeing the line on 91 of config.toml

Reply:
@rootulp Yes, just checked, we have both. Which refer to what? I guess they're the same no?

Reply:
The `client.toml` contains the default configuration for running commands, https://github.com/cosmos/cosmos-sdk/pull/8953.
So, I will revert the change: [10b99a6](https://github.com/celestiaorg/celestia-app/pull/1386/commits/10b99a639ac260f9be1bcc5cd0ea2edda2acc34b)

Reply:
Good catch, messed the address